src/get_template.py

```diff
@@ -38,7 +38,7 @@ def __get_invite_link(self):
             except requests.exceptions.ConnectionError:
                 print(f"No response is received from that link. Please try again.")
                 continue
-            if re.match(r"^https?://(www\.)?discord(app)?\.com/invite/(gg/)?.{2,}$", r.url):
+            if re.match(r"^https?://(www\.)?discord(app)?\.com/invite/?.{2,}$", r.url):
```

Comment:
```suggestion
            if re.match(r"^https?://(www\.)?discord(app)?\.com/invite/.{2,}$", r.url):
```
The optional operator also needs to be removed.

Reply:
4AM hangover : ) Fixed.